frontend/simple/views/NavBar.vue

```diff
@@ -73,7 +82,7 @@
               <i18n>Signout</i18n>
             </a>
             <div class="button" style="border: 0" v-if="$store.state.loggedIn">
-              <img src="http://bulma.io/images/placeholders/16x16.png" data-pin-nopin="true">&nbsp;<strong><i18n>Welcome</i18n>, {{$store.state.loggedIn}}</strong>
+              <img v-if="$store.getters.currentUserIdentityContract && $store.getters.currentUserIdentityContract.attributes && $store.getters.currentUserIdentityContract.attributes.picture" v-bind:src="$store.getters.currentUserIdentityContract.attributes.picture" data-pin-nopin="true">&nbsp;<strong>Welcome, {{ ($store.getters.currentUserIdentityContract && $store.getters.currentUserIdentityContract.attributes && $store.getters.currentUserIdentityContract.attributes.displayName ? $store.getters.currentUserIdentityContract.attributes.displayName : null) || $store.state.loggedIn.name}}</strong>
```

Comment:
Pinterest has a plugin for browsers that people can use to automatically upload images